### ISSUE TITLE
Official website not working

### ISSUE THREAD

**Issue by timbasel** [Lattyware/manydecks#30]
Opening the `decks.rereadgames.com` website only shows the spinner and nothing else. In the console the website also prints hundreds and hundreds of errors (see below).

I don't know if this is general or just for me, but I have tested it on 2 computers and multiple browsers, with the same error.

```
Uncaught TypeError: Cannot read property 'elmFs' of undefined
    at Un (many-decks.4400b9570fb1efc115b1.js:1)
    at xn (many-decks.4400b9570fb1efc115b1.js:1)
    at Xn (many-decks.4400b9570fb1efc115b1.js:1)
    at Gn (many-decks.4400b9570fb1efc115b1.js:1)
    at Fn (many-decks.4400b9570fb1efc115b1.js:1)
    at many-decks.4400b9570fb1efc115b1.js:1
    at e (many-decks.4400b9570fb1efc115b1.js:1)
```

**Comment by Lattyware** [comment]
Hrm, I suspect this is something like an extension blocking the script or something. That's an error happening at a really fundamental level, and I can't reproduce in anything. Could you give me details of what browsers and versions you are using on what platforms, and try disabling any extensions that might mess with the page content. Really odd you are seeing it across devices and browsers, but I'm not sure what could be causing that.

**Comment by timbasel** [comment]
Thanks for the quick reply. 
I tried it again on a freshly installed Chromium 89.0.4389.90 on Linux and everything appears to be working fine. Even with the extensions I had enabled earlier (uBlock Origin and uMatrix) it works. On Firefox 86.0.1 it is the same. So I can't replicate the error on any newly installed browser either. :thinking:

In conclusion it was definitely an issue on my site. 
PS: Thank you for the awesome game and website.
